grails-app/views/opus/_editBiostatus.gsp

```diff
@@ -0,0 +1,58 @@
+<div class="panel panel-default" ng-form="BioStatusFrom" ng-cloak>
+    <div class="panel-heading">
+        <a name="biostatus">
+            <h4>Bio Status List</h4>
+        </a>
+    </div>
+
+    <div class="panel-body">
+        <div class="col-sm-12">
+            <p>Configure the species lists to be included in your profile pages.</p>
```

Comment:
Need to change this text to explain what needs to be done. Use this:

"Configure the lists to be used for the status(es) of your profiles. These lists can include such statuses as biostatus, naturalised status, pest status, and so forth.

Using the Atlas of Living Australia Lists Tool (with a link), upload a Species List as a .csv file, where the first column is "scientific_name" (this must match the name of your profile, excluding author information), plus a separate column for each status you wish to display."